src/test/kotlin/com/seokjoo/todo/domain/service/todo/TodoServiceTest.kt

```diff
@@ -0,0 +1,184 @@
+package com.seokjoo.todo.domain.service.todo
+
+import com.seokjoo.todo.common.exception.TodoException
+import com.seokjoo.todo.common.exception.TodoExceptionType
+import com.seokjoo.todo.domain.entity.category.Category
+import com.seokjoo.todo.domain.entity.todo.Todo
+import com.seokjoo.todo.domain.entity.todocategory.TodoCategory
+import com.seokjoo.todo.domain.repository.category.CategoryRepository
+import com.seokjoo.todo.domain.repository.todo.TodoRepository
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+import io.mockk.every
+import io.mockk.mockk
+import org.springframework.data.repository.findByIdOrNull
+
+class TodoServiceTest : BehaviorSpec({
+    val todoRepository: TodoRepository = mockk()
+    val categoryRepository: CategoryRepository = mockk()
+
+    val todoService = TodoService(
+        todoRepository = todoRepository,
+        categoryRepository = categoryRepository,
+        todoCategoryRepository = mockk(),
+    )
+
+    Given("create todo") {
+        val request = TodoServiceRequestDTO(
+            todo = "abcde",
+            isDone = false,
+        )
+        val requestTodo = Todo(id = 1L, todo = request.todo, isDone = request.isDone)
+        When("when") {
+            every { todoRepository.save(any()) } returns requestTodo
+            val todo = todoService.createTodo(request)
+            Then("create Todo 시에, todo 한개가 잘 생성된다") {
+                todo.isDone shouldBe false
+                todo.todo shouldBe "abcde"
+            }
+        }
+    }
+
+    Given("getTodos") {
+        val id = 1L
+        every { todoRepository.findByIdOrNull(id) } returns Todo(id = 1L, todo = "abcde")
+        When("when") {
```

Comment:
```suggestion
        When("정상 케이스") {
```

`When("exception")`이 있으니 `When("when")`보단 요게 어떠신지 ,,

Reply:
b1c01a49a3ecbe42ba25f8e3a853766fe6eaa8ba 반영했습니당